### PR TITLE
Adds the ability to add new stories

### PR DESCRIPTION
**Pull request by mattvv**
Yeah this is a bit of fun :D

### CODE REVIEW

File: components/Column.tsx [hunk 1]
```diff
@@ -45,15 +46,15 @@ const colors = {
...
+  const { type, palette } = useTheme();
 
   return (
     <ColumnContainer
       key={idx}
       colors={colors[state]}
       background={palette.accents_2}
       shadow={palette.accents_1}>
-      <Header h5 color={palette.accents_6}>
+      <Header h5 color={type === 'light' ? palette.accents_6 : palette.accents_7}>
```
Comment (mattvv) [inline components/Column.tsx:57]: this is from the other pr #33 you can ignore for this

File: components/StoryCard.tsx [hunk 2]
```diff
@@ -18,13 +19,20 @@ const CardContainer = styled(Card)(({ color }) => ({
   borderColor: `${color} !important`,
 }));
 
-const borderColors = {
+const typeColors = {
```
Comment (mattvv) [inline components/StoryCard.tsx:22]: this is from the other pr #33 you can ignore for this

File: components/StoryCard.tsx [hunk 3]
```diff
@@ -18,13 +19,20 @@ const CardContainer = styled(Card)(({ color }) => ({
...
   feature: 'gray',
   bug: 'red',
   chore: 'green',
   release: 'blue',
 };
 
+const StoryType = styled(Breadcrumbs.Item)(({ color }) => ({
```
Comment (mattvv) [inline components/StoryCard.tsx:29]: this is from the other pr #33 you can ignore for this

File: components/StoryCard.tsx [hunk 4]
```diff
@@ -81,16 +98,15 @@ const StoryCard = ({ story, state, index, addFilter }: StoryCardParams): JSX.Ele
             <CardContainer
               width="250px"
               hoverable
-              style={{ borderColor: borderColors[story.story_type] || 'gray' }}
               onClick={e => {
-                if (e.target.nodeName === 'TEXTAREA') {
+                if (['A', 'SPAN', 'TEXTAREA'].includes(e.target.nodeName)) {
```
Comment (mattvv) [inline components/StoryCard.tsx:102]: this is from the other pr #33 you can ignore for this

**Review by jansen0825 — APPROVED** [review]
Nice!

**Review by evanpettie — APPROVED** [review]

**Pull request merged**